flavors/bioinformatics/Dockerfile

```diff
@@ -0,0 +1,316 @@
+# syntax=docker/dockerfile:1
+# MEGALINTER FLAVOR [bioinformatics]: Optimized for bioinformatics projects
+###########################################
+###########################################
+## Dockerfile to run MegaLinter ##
+###########################################
+###########################################
+
+# @not-generated
+
+#############################################################################################
+## @generated by .automation/build.py using descriptor files, please do not update manually ##
+#############################################################################################
+#ARGTOP__START
+# renovate: datasource=docker depName=rhysd/actionlint
+ARG ACTION_ACTIONLINT_VERSION=1.7.4
+# renovate: datasource=docker depName=koalaman/shellcheck
+ARG BASH_SHELLCHECK_VERSION=v0.10.0
+#ARGTOP__END
+
+#############################################################################################
+## @generated by .automation/build.py using descriptor files, please do not update manually ##
+#############################################################################################
+#FROM__START
+FROM rhysd/actionlint:${ACTION_ACTIONLINT_VERSION} AS actionlint
+# shellcheck is a dependency for actionlint
+FROM koalaman/shellcheck:${BASH_SHELLCHECK_VERSION} AS shellcheck
+# Next FROM line commented because already managed by another linter
+# FROM koalaman/shellcheck:${BASH_SHELLCHECK_VERSION} AS shellcheck
+FROM mvdan/shfmt:latest-alpine AS shfmt
+#FROM__END
+
+##################
+# Build wheel for megalinter python package
+##################
+FROM ghcr.io/astral-sh/uv:0.5.1 AS uv
+FROM python:3.12.7-alpine3.20 AS build-ml-core
```

Comment:
Once again, this warning is wrong. [GitHub's docs](https://docs.github.com/en/actions/sharing-automations/creating-actions/dockerfile-support-for-github-actions#user) say:

> Docker actions must be run by the default Docker user (root). Do not use the USER instruction in your Dockerfile, because you won't be able to access the GITHUB_WORKSPACE directory.

Does anyone know how to tweak these warnings?

Reply:
Hmm I'm not sure we have permission to tweak the wiz settings, we likely would have to talk to IT. Not sure if we already have any tickets or previous requests about tweaking the scanning? @tyamaguchi-ucla